web/source/keymanweb.js

```diff
@@ -1318,7 +1317,26 @@ if(!window['tavultesoft']['keymanweb']['initialized']) {
     keymanweb.updateInput=function(x){};
   
     // End of I3363 (Build 301) additions
-    
+
+    /**
+     * Function     resetContext
+     * Scope        Public
+     * Description  Revert OSK to default layer and clear any deadkeys and modifiers
+     */
+    keymanweb['resetContext'] = keymanweb.resetContext = function()
```

Comment:
@jahorton Can you review this function for correctness? I am a little unsure of the `osk._Visible = 0` line and also you will be more aware of any other state variables that may need reset.

Reply:
Done.  It looks correct to the best of my understanding; it'll even work correctly with the upcoming chirality changes.  `_ResetVKShift` is where that magic is.